src/alr/alr-commands-setenv.adb

```diff
@@ -14,16 +14,7 @@ package body Alr.Commands.Setenv is
 
       Requires_Valid_Session;
 
-      --  Temporarily raise the log level to avoid spurious status output that
-      --  would prevent directly sourcing the output. TODO: remove this once
-      --  the lockfile is used, that will make this unnecessary.
-      declare
-         Old_Level : constant Simple_Logging.Levels := Alire.Log_Level;
-      begin
-         Alire.Log_Level := Simple_Logging.Always;
-         Alr.Build_Env.Print (Alr.Root.Current);
-         Alire.Log_Level := Old_Level;
-      end;
+      Alr.Build_Env.Print (Alr.Root.Current);
```

Comment:
Why are you removing the temporary change of output level?

Reply:
After #385 the problem with the logging no longer happens. The detection status either goes away on regular runs or doesn't happen on redirections.

Reply:
ok :+1: